scripts/karma.conf.js

```diff
@@ -27,7 +27,6 @@ module.exports = function(config) {
     browserstackLaunchers(defaults) {
       delete defaults.bsSafariMojave;
       delete defaults.bsSafariElCapitan;
-      delete defaults.bsEdgeWin10;
       delete defaults.bsIE11Win10;
```

Comment:
did we want to enable IE11 as well like the PR title suggests?

Reply:
woops